source/kraken/tests/realtime_test.cpp

```diff
@@ -55,6 +55,35 @@ make_cancellation_message(const std::string& vj_uri, const std::string& date) {
     return trip_update;
 }
 
+/*
+template<typename T>
+struct TypeChecker;
+*/
+int64_t to_int64(const std::string& date) {
+    return (pt::from_iso_string(date) - pt::from_time_t(0)).ticks();
```

Comment:
and I don't think it's good to use ticks, it depends on the implementation of the duration, it's better to use total_seconds()